### PR TITLE
[Merged by Bors] - TY-2922 search and headlines use different market rank mappings

### PR DESCRIPTION
**Pull request by rustonaut**
(through both still use the same values for now).

### CODE REVIEW

File: discovery_engine_core/providers/src/filter.rs [hunk 1]
```diff
@@ -54,7 +54,7 @@ pub struct Market {
 }
 
 impl Market {
-    /// Returns the default news quality rank limit for given country.
+    /// Returns the default news quality rank limit for given country for search news.
```
Comment (finiteprods) [inline discovery_engine_core/providers/src/filter.rs:57]:
```suggestion
    /// Returns the default quality rank limit for news.
```

File: discovery_engine_core/providers/src/filter.rs [hunk 2]
```diff
@@ -72,6 +72,25 @@ impl Market {
             _ => return None,
         })
     }
+
+    /// Returns the default news quality rank limit for given country for latest headlines.
```
Comment (finiteprods) [inline discovery_engine_core/providers/src/filter.rs:76]:
```suggestion
    /// Returns the default quality rank limit for latest headlines.
```

**Review by finiteprods — APPROVED** [review]
included a minor suggestion for shortening the docstrings a bit otherwise looks good.

**Comment by rustonaut** [comment]
bors merge

**Pull request closed without merging**